backend/src/login/index.js

```diff
@@ -89,6 +91,7 @@ function login(strategy, req, res, next) {
   })(req, res, next)
 }
 
+/* eslint-disable complexity, max-statements */
```

Comment:
I'd prefer to pull out the third-party authentication processes (vault, sysupport) into separate functions ^_^

Reply:
yeah, makes the code more readable, and done